src/Features/RulesMissingDocumentation.md

```diff
@@ -2,27 +2,24 @@
-IDE0150 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0150> | Prefer 'null' check over type check |
-IDE0160 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0160> | Convert to block scoped namespace |
-IDE0161 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0161> | Convert to file-scoped namespace |
-IDE0170 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0170> | Property pattern can be simplified |
-IDE0180 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0180> | Use tuple to swap values |
-IDE0200 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0200> | Remove unnecessary lambda expression |
-IDE0210 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0210> | Convert to top-level statements |
-IDE0211 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0211> | Convert to 'Program.Main' style program |
-IDE0220 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0220> | Add explicit cast |
-IDE0230 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0230> | Use UTF-8 string literal |
-IDE0240 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0240> | Remove redundant nullable directive |
-IDE0241 | <https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0241> | Remove unnecessary nullable directive |
+IDE0120 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0120> | Simplify LINQ expression |
+IDE0130 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0130> | Namespace does not match folder structure |
+IDE0200 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0200> | Remove unnecessary lambda expression |
+IDE0210 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0210> | Convert to top-level statements |
+IDE0211 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0211> | Convert to 'Program.Main' style program |
+IDE0220 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0220> | Add explicit cast |
+IDE0230 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0230> | Use UTF-8 string literal |
+IDE0240 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0240> | Remove redundant nullable directive |
+IDE0241 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0241> | Remove unnecessary nullable directive |
+IDE0250 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0250> | Make struct 'readonly' |
 IDE1007 |  |  |
```

Comment:
Can we fix this entry?

Reply:
@mavasani https://github.com/dotnet/roslyn/blob/96dc83d5eecde38f9ed159e140b1256b6ce9ec2e/src/Features/Core/Portable/Diagnostics/Analyzers/UnboundIdentifiersDiagnosticAnalyzerBase.cs#L55

We want to get rid of it anyway. It requires compiler changes which I started doing in https://github.com/dotnet/roslyn/pull/62874, but from PR reviews, it was suggested that the compiler shouldn't produce `IncompleteMemberSyntax` at all, and we should obsolete it.

I haven't had enough time to complete the work since then, but hopefully I can get time after exams (ends in two days)